xbmc/settings/AdvancedSettings.cpp

```diff
@@ -849,15 +841,6 @@ void CAdvancedSettings::ParseSettingsFile(const std::string &file)
     XMLUtils::GetString(pElement, "catrustfile", m_caTrustFile);
   }
 
-  pElement = pRootElement->FirstChildElement("cache");
```

Comment:
It'd be great if instead of just removing this we could somehow migrate the settings, removing it completely by v22

Reply:
This is not trivial because not all possible values can be translated to new settings (only some fixed steps values).

It would be complicated to implement and in case of errors it could be worse. Additionally, some users have meaningless values such as Read Factor = 999 that comes from some add-ons and "wizards" advancedsettings generators :)

Reply:
I am happy with the provided solution (log)